tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1160,15 +1160,15 @@ fillSymbolInfo(CursorSymbolInfo &Symbol, const DeclInfo &DInfo,
 
 /// Returns true on success, false on error (and sets `Diagnostic` accordingly).
 static bool passCursorInfoForDecl(
-    const ResolvedValueRefCursorInfo &Info, bool AddRefactorings,
+    ResolvedValueRefCursorInfoPtr Info, bool AddRefactorings,
```

Comment:
This is the only use of `ResolvedValueRefCursorInfoPtr` right? It could theoretically stay as a `const ResolvedValueRefCursorInfo &Info` and then just stack allocate `ResolvedValueRefCursorInfo` in the two callers. But also I strongly dislike all the callbacks here, so maybe this will make it easier to get rid of that later 🤷